src/core/lib/surface/lame_client.cc

```diff
@@ -43,7 +42,7 @@ struct CallData {
   grpc_call_combiner* call_combiner;
   grpc_linked_mdelem status;
   grpc_linked_mdelem details;
-  grpc_core::atomic<bool> filled_metadata;
+  std::atomic<bool> filled_metadata;
```

Comment:
Our general rule has been that we don't use anything from `std::` outside of the src/core/lib/{gpr,gprpp,iomgr} directories.  Even if `grpc_core::atomic` is always defined as `std::atomic`, I think it's useful to have the indirection.

Reply:
Thank you for the context. I reverted this part and aliased std::atomic as grpc_core::atomic.

`src/core/ext/filters/load_reporting/server_load_reporting_filter.cc` was also using `std::atomic` which I changed to `grpc_core::atomic`